templates/resources.yml

```diff
@@ -87,16 +87,17 @@ Resources:
                 'cognito-identity.amazonaws.com:aud': !Ref CognitoIdentityPool
               'ForAnyValue:StringLike':
                 'cognito-identity.amazonaws.com:amr': authenticated
+      # TODO: Allow invoking AppSync API
       # Authenticated users are allowed to invoke the API
-      Policies:
-        - PolicyName: InvokeApi
-          PolicyDocument:
-            Version: '2012-10-17'
-            Statement:
-              - Effect: Allow
-                Action:
-                  - execute-api:Invoke
-                Resource: !Sub 'arn:aws:execute-api:${AWS::Region}:${AWS::AccountId}:${ApiGatewayRestApi}/${{self:provider.stage}}/*/*'
+      # Policies:
+      #   - PolicyName: InvokeApi
+      #     PolicyDocument:
+      #       Version: '2012-10-17'
+      #       Statement:
+      #         - Effect: Allow
+      #           Action:
+      #             - execute-api:Invoke
+      #           Resource: !Sub 'arn:aws:execute-api:${AWS::Region}:${AWS::AccountId}:${ApiGatewayRestApi}/${{self:provider.stage}}/*/*'
```

Comment:
@bboure Could you please take a look at this and replacing the Cognito Authorizer stuff below to work with AppSync?

Reply:
@brettstack I'll have a look asap.